lib/evm_database.rb

```diff
@@ -28,9 +28,8 @@ class EvmDatabase
     MiqPolicySet
     ChargebackRateDetailMeasure
     ChargeableField
-    ChargebackRateDetailCurrency
     ChargebackRate
-
```

Comment:
This blank line was here intentionally, but I can't remember why I split the list 😆  ...can you leave it, and also keep Currency in the upper section?